lib/wallet/wallet.js

```diff
@@ -1805,15 +1881,21 @@ class Wallet extends EventEmitter {
    * @param {Number} value
    * @param {Number} lockup
    * @param {Object} options
+   * @param {Event=} parentEvent
    * @returns {MTX}
    */
 
-  async createBid(name, value, lockup, options) {
-    const unlock = await this.fundLock.lock();
+  async createBid(name, value, lockup, options, parentEvent) {
```

Comment:
nit: would prefer `parentEvent = this.honey` in the function signature to set a default so we don't need to do the `parentEvent || this.honey` which is slightly less obviously a default value. (note: there are several instances of this)